plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -68,10 +73,33 @@
 public class Maven2GroupFacet
     extends GroupFacetImpl
 {
+  @VisibleForTesting
+  static final String CONFIG_KEY = "maven2group";
+
+  @VisibleForTesting
+  static class Config
```

Comment:
This is just following the pattern we have else where that a Facet has a Config object to bind/validate its configuration to a POJO.  No need to diverge from that IMO.